piptools/scripts/compile.py

```diff
@@ -164,6 +164,12 @@ def cli(
     """
     log.verbosity = verbose - quiet
 
+    # If `src-files` was not provided as input, but rather as config,
+    # it will be part of the click Context `ctx`.
+    # However, is `src_files` is specified, then we want to use that.
```

Comment:
We generally use RST syntax, I think consistency with the comments would be good.

```suggestion
    # If ``src-files` was not provided as an input, but rather as config,
    # it will be part of the click context ``ctx``.
    # However, is ``src_files`` is specified, then we want to use that.
```